### PR TITLE
Fix #39

### LINKED ISSUE #39: What section should have dot background

### PR DESCRIPTION
**Pull request by mailtime-wing**
Add dot background in what section. Resolve #39

### CODE REVIEW

File: src/components/SlopedBackground/index.js [hunk 1]
```diff
@@ -9,23 +10,46 @@ const SlopedSection = ({
...
-    {...props}
-  >
-    {children}
-  </BackgroundWithNavigator>
-)
+}) => {
+  const theme = useTheme()
+  return (
+    <BackgroundWithNavigator
+      css={css`
```
Comment (chochihim) [inline src/components/SlopedBackground/index.js]: this `useTheme` is not necessary. The `css` props can be a function with theme as argument.

Comment (mailtime-wing) [reply src/components/SlopedBackground/index.js]: fixed.

**Comment by chochihim** [comment]
@mailtime-wing This looks good. Can you rebase to master and resolve conflict?

**Comment by mailtime-wing** [comment]
Rebased and resolved all the conflict

**Pull request merged**